.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,8 @@
 
 ## Why?
 
-### Before
+## Screenshots
 
-### After
+| **Before** | **After** |
+|------------|-----------|
+| Image      | Image     |
```

Comment:
Oh wow, I didn't know you could do that. I like this approach, if we do this can we add a comment that you just need to replace the url's and not actually modify the table?

Reply:
Named links, works with standard links too. When writing cited docs, I use it a lot.

e.g. from [Daring fireball][daringfireball]
```markdown
I get 10 times more traffic from [Google][1] than from
[Yahoo][2] or [MSN][3].

[1]: http://google.com/        "Google"
[2]: http://search.yahoo.com/  "Yahoo Search"
[3]: http://search.msn.com/    "MSN Search"
```

[daringfireball]: https://daringfireball.net/projects/markdown/basics "Daring fireball"

Reply:
Nice suggestion on the comment. I found it easier as GH adds the link where your cursor is when uploading images.